tests/test_component.py

```diff
@@ -439,6 +440,129 @@ def test_can_add_mesh_shape_to_and_component_and_get_the_same_shape_back(nexus_w
     assert output_mesh.vertices[2].z() == approx(vertex_2_z)
 
 
+def test_can_get_cad_file_units_from_file(nexus_wrapper):
```

Comment:
This test can be removed, I don't think it is testing anything not covered in the other tests.